ossindex/ossindex.go

```diff
@@ -106,68 +108,88 @@ func AuditPackages(purls []string) ([]types.Coordinate, error) {
 		return nil, err
 	}
 
-	if len(newPurls) > 0 {
-		var request types.AuditRequest
-		request.Coordinates = newPurls
-		var jsonStr, _ = json.Marshal(request)
+	var chunks = chunk(newPurls, MAX_COORDS)
 
-		req, err := setupRequest(jsonStr)
-		if err != nil {
-			return nil, err
-		}
+	for _, chunk := range chunks {
+		if len(chunk) > 0 {
+			var request types.AuditRequest
+			request.Coordinates = chunk
+			var jsonStr, _ = json.Marshal(request)
 
-		client := &http.Client{}
-		resp, err := client.Do(req)
-		if err != nil {
-			return nil, err
-		}
+			req, err := setupRequest(jsonStr)
+			if err != nil {
+				return nil, err
+			}
 
-		switch {
-		case strings.Contains(resp.Status, "200"):
-			fmt.Println(resp)
-		default:
-			return nil, errors.New("[" + resp.Status + "] error accessing OSS Index")
-		}
+			client := &http.Client{}
+			resp, err := client.Do(req)
+			if err != nil {
+				return nil, err
+			}
 
-		defer func() {
-			if err := resp.Body.Close(); err != nil {
-				log.Printf("error closing response body: %s\n", err)
+			switch {
+			case strings.Contains(resp.Status, "200"):
```

Comment:
SUGGESTION: Pretty sure you can do: `resp.StatusCode != http.StatusOK` to have a cleaner reading line and to avoid using strings.Contains.

See: 
https://golang.org/src/net/http/response.go

https://golang.org/src/net/http/status.go

Reply:
Since you only have one case as well, using an `if else` line would likely be better than using a `switch`